src/app/Http/Controllers/Admin/SessionController.php

```diff
@@ -41,7 +41,7 @@ public function index()
                     ->with([
                         'owner',
                         'testCases' => function ($query) {
-                            return $query->with(['lastTestRun']);
+                            return $query->with(['useCase']);
```

Comment:
I'm a little bit nervous about this change, unless you are 100% confident that `testCases[*].lastTestRun` is never ever used on the front end (e.g. I haven't looked in detail, but a quick Ctrl+F suggests it might used [here](https://github.com/gsmainclusivetechlab/interop-test-platform/blob/master/src/resources/js/components/sessions/progress.vue#L29)?).

I think you're right that we do need to retrieve the useCases, but I think let's not remove `lastTestRun` at the same time:
```suggestion
                            return $query->with(['useCase', 'lastTestRun']);
```

or even
```suggestion
                            return $query->with(['useCase:id', 'lastTestRun']);
```

[Supporting Documentation](https://laravel.com/docs/7.x/eloquent-relationships#eager-loading) (subheading "Eager Loading Multiple Relationships" and "Eager Loading Specific Columns")